assets/js/index.js

```diff
@@ -0,0 +1,13 @@
+function menu() {
+  const humberger = document.getElementById('humberger');
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223351904-6a795754-2a43-4005-aace-0adf5d7ff1cb.png)

- [ ] It would be nice if you don't repeat your code to make it clean and to follow the JavaScript best practices :+1: At the bottom of the image you can see DRY means Don't repeat yourself :+1:

Line of code you have repeated :arrow_down:
**const humberger = document.getElementById('humberger');**

Reply:
Thanks for feedback
Eslint is showing errors whenever I try to make it so. do you think that is an error to leave it like this ? @HAMMAS-SALEEM